pool/pool.py

```diff
@@ -711,21 +711,22 @@ async def get_and_validate_singleton_state(
         # Validate state of the singleton
         is_pool_member = True
         if singleton_tip_state.target_puzzle_hash != self.default_target_puzzle_hash:
-            self.log.info(f"Wrong target puzzle hash: {singleton_tip_state.target_puzzle_hash}")
+            self.log.info(f"Wrong target puzzle hash: {singleton_tip_state.target_puzzle_hash} for launcher_id {launcher_id}")
             is_pool_member = False
         elif singleton_tip_state.relative_lock_height != self.relative_lock_height:
-            self.log.info(f"Wrong relative lock height: {singleton_tip_state.relative_lock_height}")
+            self.log.info(f"Wrong relative lock height: {singleton_tip_state.relative_lock_height} for launcher_id {launcher_id}")
             is_pool_member = False
         elif singleton_tip_state.version != POOL_PROTOCOL_VERSION:
-            self.log.info(f"Wrong version {singleton_tip_state.version}")
+            self.log.info(f"Wrong version {singleton_tip_state.version} for launcher_id {launcher_id}")
             is_pool_member = False
         elif singleton_tip_state.state == PoolSingletonState.SELF_POOLING.value:
-            self.log.info(f"Invalid singleton state {singleton_tip_state.state}")
+            self.log.info(f"Invalid singleton state {singleton_tip_state.state} for launcher_id {launcher_id}")
             is_pool_member = False
         elif singleton_tip_state.state == PoolSingletonState.LEAVING_POOL.value:
             coin_record: Optional[CoinRecord] = await self.node_rpc_client.get_coin_record_by_name(singleton_tip.coin)
             assert coin_record is not None
             if self.blockchain_state["peak"].height - coin_record.confirmed_block_index > self.relative_lock_height:
+                self.log.info("Removing launcher_id {launcher_id} because they have left the pool")
```

Comment:
```suggestion
                self.log.info(f"launcher_id {launcher_id} got enough confirmations to leave the pool")
```

There is a `f` missing and how about changing the text (i'm open for other wording like the suggested ones :slightly_smiling_face:) because technically it still didn't leave the pool, right? And also we don't remove anything internally if someone leaves.

Reply:
Yes, good points, thank you!